detectron2/data/datasets/coco.py

```diff
@@ -280,6 +286,131 @@ def file2id(folder_path, file_path):
     return dataset_dicts
 
 
+def convert_to_coco_dict(dataset_name):
+    """
+    Convert a generic dataset into COCO json format
```

Comment:
"Convert a dataset in detectron2's standard format into COCO json format"

"Detectron2's standard dataset format description can be ..."

Reply:
done